ambry-quota/src/main/java/com/github/ambry/quota/AmbryQuotaManager.java

```diff
@@ -122,14 +122,19 @@ public ThrottlingRecommendation getThrottleRecommendation(RestRequest restReques
   @Override
   public ThrottlingRecommendation charge(RestRequest restRequest, BlobInfo blobInfo,
       Map<QuotaName, Double> requestCostMap) {
+    return charge(restRequest);
```

Comment:
This method should probably stay for future usage.

Reply:
The main implementation for this interface is in closed source code. The method exists there.